.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+   "tabWidth": 3,
```

Comment:
This should be 2

Reply:
oops, forgot to change it back after testing